### ISSUE TITLE
Transactions off, but no error

### ISSUE THREAD

**Issue by particularbot** [Particular/NServiceBus#3139]
**Issue by [johnsimons](https://github.com/johnsimons)** _11/23/2015 3:01:06 AM +00:00_ _Originally opened as https://github.com/Particular/ServiceInsight/issues/511_

----

I have the following diagram:

![image](https://cloud.githubusercontent.com/assets/122651/11329042/71c55c76-91e1-11e5-89bb-16bec5e608f9.png)

As you can see Subscriber1 is sending 2 messages out but that processing handler is not connected to an incoming message.
So to reproduce this, I have disabled transactions:
```c#
Configure.Transactions.Disable();
Configure.Transactions.Advanced(a => a.DoNotWrapHandlersExecutionInATransactionScope().DisableDistributedTransactions());
```

and my handler looks like:
```c#
public void Handle(AnotherEventMessage message)
{
    // So these messages are going out
    bus.SendLocal(new ThisShouldGoOut());
    bus.Publish<MySuperPowerAreEnabled>();

    throw new Exception("Something is off");
}
```

The thing is missing in the diagram is an error message, but it seems this is by design in the core :disappointed:

**Comment by particularbot** [comment]
**Comment by [johnsimons](https://github.com/johnsimons)** _https://github.com/Particular/ServiceInsight/issues/511#issuecomment-158839344_

----

@andreasohlund @danielmarbach @SzymonPobiega @timbussmann  would really appreciate some thoughts here ?

**Comment by johnsimons** [comment]
BTW, had a go at upgrading this sample to v6 and here is the same scenario running v6:
![image](https://cloud.githubusercontent.com/assets/122651/11329357/c7cf3a4c-91e6-11e5-8dea-e6b92f5c630e.png)

**Comment by andreasohlund** [comment]
Is this a duplicate of #3130 ?

**Comment by WojcikMike** [comment]
With @andreasohlund we verified it and below are the findings. In v5 with transations disabled the error message will not be seen, and this is by design.

When it comes to v6 my sample shown deduplication error in NSB core and after fixing it I was able to verify the sample agains v6 and everything works well (beside the fact that elements on diagrams are in wierd order but I will create separate issue around that). 

V6 Without transations:
![image](https://cloud.githubusercontent.com/assets/604826/12195637/ecc0310e-b5f8-11e5-8fc5-f42da6697ffe.png)


V6 With transations:
![image](https://cloud.githubusercontent.com/assets/604826/12195609/c050eeba-b5f8-11e5-9716-bcc1029d4ade.png)


V6 With transaton no exception thrown:
![image](https://cloud.githubusercontent.com/assets/604826/12195597/9ff1173a-b5f8-11e5-9f29-8719903a937f.png)

With the above verification in place I suggest closing this issue, unless @andreasohlund and @johnsimons suggest otherwise
